R/LSTbinomDist.R

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2019 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+LSTbinomDist <- function(jaspResults, dataset, options) {
```

Comment:
So what is actually the end goal here? Do we want to have the functionality exactly the same as in `jaspDistributions`, or is there something that we need to do differently?

If the former is the case, then I would suggest simply

```suggestion
LSTbinomDist <- jaspDistributions::LDbinomial
```
then we know that whatever `jaspDistributions` do the LearnStats module also does.

The second part would be to import the QML forms from `jaspDistributions` into this module. This requires adding `qmldir` file in `jaspDistributions` (https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/jasp-qml-guide.md#custom-imports) and just reuse the form.